packages/office-ui-fabric-react/src/utilities/color/colors.ts

```diff
@@ -156,7 +170,24 @@ export function getColorFromString(color: string): IColor {
     str: color,
     v: v
   };
+}
 
+export function getColorFromRGBA(rgba: { a: number, b: number, g: number, r: number }): IColor {
```

Comment:
Nit: rather than have the properties in alphabetical order (abgr), can you just have them in rgba order?